client/networkmanager/Makefile

```
@@ -0,0 +1,80 @@
+	@mkdir -p bin
+	$(GO) build -o $@ ./cmd/nm-netbird-auth-dialog
+
+ifeq ($(SKIP_PROPERTIES),1)
+properties:
+	@echo "skipping client/networkmanager/properties (SKIP_PROPERTIES=1)"
+else
+properties:
+	$(MAKE) -C properties
+endif
+
+install: all
```

checkmake (0.3.2)

[warning] Target body for "install" exceeds allowed length of 5 lines (16).

(maxbodylength)